.github/workflows/future-release-branch-labeler.yml

```
@@ -0,0 +1,43 @@
+name: Future Release Branch Pull Request Labeler
+
+on:
+  pull_request_target:
+    branches:
+      - dev-[0-9].[0-9][0-9]
+    types:
+      - opened
+      - reopened
+
+jobs:
+  add-hold-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+          curl --request PATCH \
+          --url ${{ github.event.pull_request.issue_url }} \
+          --header 'Authorization: token ${{ secrets.GITHUB_ACTIONS_TOKEN }}' \
+          --header 'Accept: application/vnd.github.v3+json' \
+          --data '{"labels": ["hold"]}'
+  add-milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+          # grab the version using the branch of the base repository
```

Is this accurate:
```suggestion
          # determine the target release from the branch that this PR targets
          # ignore first 4 characters ("dev-")
```
?